Updated course details on sem 4

gauthamalt opened the pull request.

its-dev24 committed:
Updated course details on sem 4

techworldthink reviewed (commented):
Please provide a description of the changes you have made.

gauthamalt commented:
So basically what i have done is just the typo changes .. The description of sem 4 was like abcd and it was apecified as web developement i had updated it to be Main project and just updated the description
